### PR TITLE
Change username URL

### PR DESCRIPTION
**Pull request by NuclearPhoenixx**
Changed my username from Phoenix1747 to NuclearPhoenixx. Updated the repo URLs to reflect that change.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @NuclearPhoenixx. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/08/14 09:19:46 Loaded 6083 libraries from DB
Changing URL of library MQ135 from https://github.com/Phoenix1747/MQ135.git to https://github.com/NuclearPhoenixx/MQ135.git
Success!
```
```
2023/08/14 09:20:28 Loaded 6083 libraries from DB
Changing URL of library PicoAnalogCorrection from https://github.com/Phoenix1747/Arduino-Pico-Analog-Correction.git to https://github.com/NuclearPhoenixx/Arduino-Pico-Analog-Correction.git
Success!
```
```
2023/08/14 09:20:57 Loaded 6083 libraries from DB
Changing URL of library SimpleShell Enhanced from https://github.com/Phoenix1747/SimpleShell.git to https://github.com/NuclearPhoenixx/SimpleShell.git
Success!
```

**Pull request merged**
